tests/transaction.test.ts

```
@@ -0,0 +1,102 @@
+
+    vi.mocked(getWalletClient).mockResolvedValue(mockWalletClient);
+    vi.mocked(getPublicClient).mockReturnValue(mockPublicClient);
+
+    const walletClient = await getWalletClient(mockRainbowKitConfig);
+    const hash = await walletClient?.sendTransaction(txParams);
+
+    expect(hash).toBe("0x456");
+
+    const publicClient = getPublicClient(mockRainbowKitConfig);
+    await expect(
+      publicClient.waitForTransactionReceipt({ hash: hash! })
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)